answers.php

```diff
@@ -1,9 +1,9 @@
 <?php
 /**
  * @wordpress-plugin
- * Plugin Name:   Yext Answers
- * Description:   Your customers have questions. Can your website answer them? The Yext Answers plugin allows you to seamlessly integrate your Yext Answers search bar and search results page with your website.
- * Version:       1.0.1
+ * Plugin Name:   Yext Answers Site Search
```

Comment:
Reading that this will change the name in the UI (which is good) but it won't change the url slug in the WP plugin store. Is this OK?

https://wordpress.stackexchange.com/questions/224205/change-plugin-name-on-wordpress-repo

Reply:
yes I think the slug is permanent: https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#can-i-change-my-plugins-urlslug